From f7232006181c15d53a70a1c7a4fb142b6c9abec2 Mon Sep 17 00:00:00 2001
From: Huisong Li <lihuisong@huawei.com>
Date: Mon, 20 Nov 2023 19:14:03 +0800
Subject: [PATCH 406/410] doc: update features in hns3 guide

[ upstream commit 97d443f751441c71f8307aea9d9ac2752fa1045c ]

Update the features for hns3.

- "Basic stats" and "Extended stats" supported by
Fixes: 8839c5e202f3 ("net/hns3: support device stats")

- "Traffic Management API" supported by
Fixes: c09c7847d892 ("net/hns3: support traffic management")

- "Speed capabilities" supported by
Fixes: 09e0de1f411b ("net/hns3: report speed capability for PF")

- "Link Auto-negotiation" supported by
Fixes: bdaf190f8235 ("net/hns3: support link speed autoneg for PF")

- "Link flow control" supported by
Fixes: 62e3ccc2b94c ("net/hns3: support flow control")

- "Dump private info from device" supported by
Fixes: 1a03c659cb9d ("net/hns3: dump device basic info")

- "FW version" supported by
Fixes: 1f5ca0b460cd ("net/hns3: support some device operations")

Cc: stable@dpdk.org

Signed-off-by: Huisong Li <lihuisong@huawei.com>
Acked-by: Jie Hai <haijie1@huawei.com>
---
 doc/guides/nics/hns3.rst | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/doc/guides/nics/hns3.rst b/doc/guides/nics/hns3.rst
index e3e6d3fbd8..d55c0930a2 100644
--- a/doc/guides/nics/hns3.rst
+++ b/doc/guides/nics/hns3.rst
@@ -30,7 +30,6 @@ Features of the HNS3 PMD are:
 - DCB
 - Scattered and gather for TX and RX
 - Vector Poll mode driver
-- Dump register
 - SR-IOV VF
 - Multi-process
 - MAC/VLAN filter
@@ -38,6 +37,15 @@ Features of the HNS3 PMD are:
 - NUMA support
 - Generic flow API
 - IEEE1588/802.1AS timestamping
+- Basic stats
+- Extended stats
+- Traffic Management API
+- Speed capabilities
+- Link Auto-negotiation
+- Link flow control
+- Dump register
+- Dump private info from device
+- FW version
 
 Prerequisites
 -------------
-- 
2.33.0

